cfpb/cfgov-refresh
# Form for the Submit A Request FOIA page

Added the Submit A Request page

## Additions

- Added a lot of markup for the submit a request page
- Created a small function to show additional fields when a certain input is selected.
- Added a utility: `u-reset` to get rid of additional formatting around the fieldset element that's added by normalize.css
- Added `input__long` to have input fields fill the container they are given.

## Testing

- Visit /initiatives/submit-a-request/
- Select "Yes" on "Requesting a fee waiver"
- Both "Mail" and "Fax" in "requested response method" should also trigger more fields to open.

## Review

- @sebworks 
- @jimmynotjim 
- @anselmbradford 

## Preview

![image](https://cloud.githubusercontent.com/assets/1860176/7868500/88f78d66-054a-11e5-91c7-88aec1c28fbe.png)

[Preview this PR without the whitespace changes](?w=0)

## Notes

- This form doesn't submit to anything for this sprint
